assets/chef-recipes/cookbooks/user_creation/recipes/default.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+user node['user_creation']['name'] do
+  shell '/bin/bash'
+end
+
+directory "/home/#{node['user_creation']['name']}" do
```

Comment:
Use [`File.join`](https://ruby-doc.org/core-2.7.1/File.html#method-c-join) instead of string interpolation to form paths.